src/main/manager/lifecycle-manager.service.ts

```diff
@@ -49,15 +49,18 @@ export class LifecycleManager {
    * If the mod is enabled, it will be disabled
    * If the mod is disabled, it will be enabled
    */
-  async toggleMod(modId: string): Promise<void> {
+  async toggleMod(modId: string, enableOnly: boolean | undefined): Promise<boolean> {
```

Comment:
Looks good overall but what does the enableOnly flag do?

Reply:
It forces the toggle to only enable to avoid the behaviour of disabling a dependency if it has already enabled when the user enables a mod using that dependency.